test/unit/CodeGenerator/Definitions/RequestParametersDtoDefinitionTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OnMoon\OpenApiServerBundle\Test\Unit\CodeGenerator\Definitions;
+
+use OnMoon\OpenApiServerBundle\CodeGenerator\Definitions\PropertyDefinition;
+use OnMoon\OpenApiServerBundle\CodeGenerator\Definitions\RequestParametersDtoDefinition;
+use OnMoon\OpenApiServerBundle\Specification\Definitions\Property;
+use PHPUnit\Framework\Assert;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \OnMoon\OpenApiServerBundle\CodeGenerator\Definitions\RequestParametersDtoDefinition
+ */
+final class RequestParametersDtoDefinitionTest extends TestCase
```

Comment:
This test is not needed, RequestParametersDtoDefinition is an empty class and DtoDefinition has its own test.